### ISSUE TITLE
\capo and own title-template

### ISSUE THREAD

**Issue by Wheart** [cgnieder/leadsheets#7]
I want to incorporate information about use of capodaster (and write song text with needed transposition). Since transposition works just OK, there is a problem with \capo command used in title-template. If command \capo is used within song environment - it works OK, but placed in tittle-template - does strange things (just prints "*" if used with capo property checking or "Capo: . fret" otherwise).

Am I doing something against Author's intention or is it a bug?

**Comment by cgnieder** [comment]
Thanks. This will be fixed in the next release. MWE:

```latex
\documentclass{article}

\usepackage{leadsheets}
\definesongtitletemplate{custom}{%
  \section*{\songproperty{title}}
  \capo
}
\setleadsheets{title-template = custom}

\begin{document}

\begin{song}{title=A}
\end{song}

\begin{song}{title=B,capo=1}
\end{song}

\begin{song}{title=C,capo=3}
\end{song}

\end{document}
```

**Comment by cgnieder** [comment]
fixed in commit f9952624de334d919676c1068372e6b422cf8c91
